idd/Energy+.idd.in

```diff
@@ -33441,7 +33449,14 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
   A11,  \field Cooling Coil Object Type
         \type choice
         \key Coil:Cooling:DX:VariableRefrigerantFlow
+        \key Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl
         \note Cooling Coil Type must be Coil:Cooling:DX:VariableRefrigerantFlow
+        \note if AirConditioner:VariableRefrigerantFlow is used 
+        \note to model VRF outdoor unit
+        \note Cooling Coil Type must be 
+        \note Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl
+        \note if AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl 
+        \note is used to model VRF outdoor unit
         \note This field may be left blank if heating-only mode is used
   A12,  \field Cooling Coil Object Name
         \type object-list
```

Comment:
\object-list CoolingCoilsDXVarRefrigFlowFluidTemperatureControl is added in IDD.